.gitignore

```diff
@@ -5,3 +5,4 @@ docker/db
 !docker/db/.gitkeep
 config/development/*
 config/envConfig.js
+/.idea
```

Comment:
.git/info/exclude would be a better home for this if possible as it's more of a personal IDE less of a project file